fenix/views/firefox_android_clients.view.lkml

```diff
@@ -0,0 +1,124 @@
+include: "//looker-hub/fenix/views/firefox_android_clients.view.lkml"
+
+view: +firefox_android_clients {
```

Comment:
What do you think about consolidating the Play Store referrer data into a single label w/ multiple items, and same with Adjust attribution?

Like this, but instead of the group label being `Metrics`, call it `Adjust`
<img width="255" alt="Screenshot 2024-03-05 at 9 26 13 AM" src="https://github.com/mozilla/looker-spoke-default/assets/20819040/23615397-9e0a-41fc-b456-a7875dbdfd96">

Reply:
That's a good suggestion, I'm not in the habit of working with Looker so I even forgot about that being an option. I will make that change prior to merging the changes.

Reply:
@fbertsch Can you take another look and let me know if this is fine:
<img width="314" alt="image" src="https://github.com/mozilla/looker-spoke-default/assets/42538694/46fae88e-1a33-4d2f-8c17-aa40cc17f9cf">